drivers/config.go

```
@@ -0,0 +1,86 @@
+
+func (s DatabaseConfiguration) Neo4jConnectionString() string {
+	if s.Connection == "" {
+		return fmt.Sprintf("neo4j://%s:%s@%s/%s", s.Username, s.Secret, s.Address, s.Database)
+	}
+
+	return s.Connection
+}
+
+func (s DatabaseConfiguration) LookupEndpoint() string {
+	host := s.Address
+	if hostCName, err := net.LookupCNAME(s.Address); err != nil {
```

golangci-lint (2.11.4)

[error] net.LookupCNAME must not be called. use (*net.Resolver).LookupCNAME with a context

(noctx)